Testing/functional/tests/cypress/support/functions/intercept.js

```diff
@@ -0,0 +1,101 @@
+const defaultHdid = "P6FFO433A5WPMVTGM7T4ZVWBKCSVNAYGTWTU3J2LWMGUMERKI72A";
+const defaultPatientFixture = "PatientService/patientCombinedAddress.json";
+const defaultUserProfileFixture = "UserProfileService/userProfile.json";
+const defaultUserProfileStatusCode = 200;
+
+export const CommunicationType = {
+    Banner: 0,
+    InApp: 2,
+};
+
+export const CommunicationFixture = {
+    Banner: "CommunicationService/communicationBanner.json",
+    InApp: "CommunicationService/communicationInApp.json",
+};
+
+/*
+----------------------------------------------------------------------------------------------------
+Usage for setupStandardIntercepts(options = {})
+
+Usage: setupStandardIntercept();
+Usage: setupStandardIntercept({ <PatientHdid> });
+Usage: setupStandardIntercept({ <USER_PROFILE_HDID>: <UserProfileHdid> });
+Usage: setupStandardIntercept({ <USER_PROFILE_STATUS_CODE>: <UserProfileStatusCode> });
+Usage: setupStandardIntercept({ <SERVICE>Fixture: <ServiceFixture> });
+Usage: setupStandardIntercept({ <IDENTIFIER>: <Identifier>, <SERVICE>Fixture: <ServiceFixture> });
+----------------------------------------------------------------------------------------------------
+*/
+export function setupStandardIntercepts(options = {}) {
+    const {
+        patientHdid = defaultHdid,
+        userProfileHdid = defaultHdid,
+        patientFixture = defaultPatientFixture,
+        userProfileFixture = defaultUserProfileFixture,
+        userProfileStatusCode = defaultUserProfileStatusCode,
+    } = options;
```

Comment:
Documentation contains typos: `setupStandardIntercept` instead of `setupStandardIntercepts`.  I'm also not sure I'm following the example usages.  I think the placeholder parameter names may be confusing things.

e.g.

```Usage: setupStandardIntercept();
Usage: setupStandardIntercepts({ patientFixture: <FixturePath> });
Usage: setupStandardIntercepts({ userProfileFixture: <FixturePath> });
Usage: setupStandardIntercepts({ userProfileStatusCode: <StatusCode> });
Usage: setupStandardIntercepts({ patientHdid: <Hdid>, patientFixture: <FixturePath> });
Usage: setupStandardIntercepts({ userProfileHdid: <Hdid>, userProfileFixture: <FixturePath> });
Usage: setupStandardIntercepts({ userProfileHdid: <Hdid>, userProfileStatusCode: <StatusCode> });
etc.
```

Reply:
Fixed.